event.py

```diff
@@ -0,0 +1,67 @@
+from copy import deepcopy
+from datetime import datetime, time
+from flask_restful import abort
+
+from eventdao import EventDAO
+
+DAYS = set(('Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'))
+
+def read_time(time_str):
+    return datetime.strptime(time_str, '%H:%M:%S').time()
+
+class Event:
+    dao = EventDAO()
+
+    def __init__(self, documnet: dict):
+        self.name = documnet['name'] if 'name' in documnet else None
+        self.days = documnet['days'] if 'days' in documnet else None
+        self.from_time = documnet['from_time'] if 'from_time' in documnet else None
+        self.to_time = documnet['to_time'] if 'to_time' in documnet else None
+        self.description = documnet['description'] if 'description' in documnet else None
```

Comment:
the best part of this `documnet` is that you not only spelled it wrong the first time, but then you proceeded to keep spelling it wrong throughout the entire constructor 👍

Reply:
copy paste man...